Vizzuality/half-earth-v3
# Feature/analyze areas sidebar card

## Analyse areas sidebar card (1st iteration)
### Description
This PR adds the `analyse areas` card to the data globe sidebar. Being the first iteration, some functionality will be addressed on following PRs. Find here a list of the things already implemented:
- [x] Pre-calculated features  dropdown selector (the layers displayed and styles are not definitive)
-  [x] Feature Highlight layer (it takes care of the interactions with the featured layers on the map. On click we would need to add the tooltip display logic. This will be handled on an standalone component)
- [x] Sketch functionality (This is the only entry point that allows a full user flow that triggers the analysis for the drawn area)
- [x] AoI hash ID generation and storing on the arcgis online service (the logic for when to store the items would need to be revised).
- [x] All taxa geo-processing services are in place and working (further refactoring of the code would be needed and also getting the data from the look up tables)
 
### Designs
[UI kit](https://www.figma.com/file/TLCNyRujxrUEhpCkNOXA6s/HalfEarth_Map_visual_V02?node-id=81%3A4332)
### Testing instructions
Go into the data globe, on the analyze areas card, select from the dropdown the different available pre-calculated layers (hover/click on the features).
Click on `draw or upload shape` button, click on start drawing, close the shape you're drawing and get transported to your area of interest. Check the dev console to see the data provided by the geo processing services
